docs/source/support/compatibility.rst

```diff
@@ -14,7 +14,7 @@ Network Upgradeability
 
   Ledger Operators should be able to upgrade Daml network or Participant Nodes seamlessly to stay up to date with the latest features and fixes. A Daml application should be able to operate without significant change across such Network Upgrades.
 
-Daml Connect Upgradeability
+Daml Upgradeability
```

Comment:
Should there be the markup for a header here?

Reply:
I don't think so, it looks intentional on [the page](https://docs.daml.com/support/compatibility.html).  There are three terms in that section, none of which have headers.